utils/hare-shutdown

```diff
@@ -55,13 +55,9 @@ def _get_processes_for_node() -> Dict[str, List[utils.Process]]:
     return processes
 
 
-def stop_process(processes: Dict[str, List[utils.Process]]):
+def stop_consul(processes: Dict[str, List[utils.Process]], leader_node: str):
```

Comment:
For me this name is misleading: we're stopping not only consul but all the processes.

`stop_all_with_consul()` maybe?

PS: that's why I like classes and overriden methods. Although the class name would be different (e.g. MiniProvFriendlyStopper and CommonStopper) but the main method would be named `stop()` in both cases 😀

Reply:
Done